### PR TITLE
Add tikv store limit pattern

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Add tikv store limit pattern which would avoid the error happen if the external store join the tikv cluster like tiflash.

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/manager/member/tikv_member_manager.go [hunk 1]
```diff
@@ -40,7 +40,8 @@ import (
 
 const (
 	// tikvClusterCertPath is where the cert for inter-cluster communication stored (if any)
-	tikvClusterCertPath = "/var/lib/tikv-tls"
+	tikvClusterCertPath   = "/var/lib/tikv-tls"
+	tikvStoreLimitPattern = `%s-tikv-\d+\.%s-tikv-peer\.%s\.svc\:\d+`
```
Comment (aylei) [inline pkg/manager/member/tikv_member_manager.go]: it is nearly hardcoded, is there a better way? or can we add a TODO that indicates we need a better way in the future?

Comment (Yisaer) [reply pkg/manager/member/tikv_member_manager.go]: This method has been used before in this request : https://github.com/pingcap/tidb-operator/pull/1621
But I'm ok to add comment to indicate find a better way in the future.

Comment (Yisaer) [reply pkg/manager/member/tikv_member_manager.go]: updated.

**Comment by Yisaer** [comment]
@aylei @cofyc  PTAL

**Pull request merged**